mkdocs-percona.yml

```diff
@@ -15,9 +15,4 @@ extra_css:
   - css/version-select.css
   - css/toctree.css
   
-markdown_extensions:
```

Comment:
Are those extensions caused yaml issues ? could you explain a little why we added them and now removing?

Reply:
I've added the extension to create tabbed note. This topic includes long chunks of code that (especially with the upcoming changes to this topic) would prove really useful to structure the document. Tabbed notes looked really nice in Render preview. However, when the topic was live, the CSS for the Percona Documentation website doesn't support this. 
Here's a video that illustrates this better https://www.loom.com/share/eab1c94121954ab8a08e0c81cc9967a2